BasicSample/Assets/AppRemoting/Scripts/AppRemotingSample.cs

```diff
@@ -133,15 +165,41 @@ public void ConnectToRemote(string address = null)
             StartCoroutine(Remoting.AppRemoting.Connect(remotingConfiguration));
         }
 
+        /// <summary>
+        /// Listens to the incoming connections as specified in the remotinglistenconfiguration
+        /// </summary>
+        public void ListenToRemote()
+        {
+            m_appRemotingMode = AppRemotingMode.listen;
+            StartCoroutine(Remoting.AppRemoting.Listen(remotingListenConfiguration));
+        }
+
         /// <summary>
         /// Disconnects from the remote session.
         /// </summary>
         public void DisconnectFromRemote()
         {
             Remoting.AppRemoting.Disconnect();
+            if (m_appRemotingMode == AppRemotingMode.listen)
+            {
+                m_disconnectedOnListenMode = true;
+            }
             ShowConnection2DUI();
         }
 
+        private string GetLocalIPAddress()
+        {
+            var host = Dns.GetHostEntry(Dns.GetHostName());
+            foreach (var ip in host.AddressList)
+            {
+                if (ip.AddressFamily == AddressFamily.InterNetwork)
```

Comment:
this returns the first address, which can be random.  I found this display the wrong ip address on my computer.

this answer on stack overflow might work better: https://stackoverflow.com/a/40528818

Reply:
thanks for the thread Oliver. The reason I left it as is , is because the IP displayed can be still used to connect on listen mode.